docker/worker/kvm-mknod.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 # If possible, create the /dev/kvm device node.
+kvm=$([[ -f /proc/config.gz ]] && test "$(gzip -c /proc/config.gz | grep CONFIG_KVM=y)")
```

Comment:
The comment is misleading. This line doesn't create anything. It just sets the KVM variable depending on the contents of `/proc/config.gz`. I would rather append ` || true` to the `grep` command instead of moving the whole line before `set -e`.

Reply:
Done